mona_sdk/client.py

```diff
@@ -328,6 +337,19 @@ def export_batch(
             events, default_action, filter_none_fields=filter_none_fields
         )
 
+    def _should_add_message_to_sampled_data(self, message):
+        context_class = message.get(CONTEXT_CLASS_FIELD_NAME)
+        context_class_sampling_rate = self._context_class_to_sampling_rate.get(
+            context_class
+        )
+        context_id = message.get(CONTEXT_ID_FIELD_NAME)
+        if context_id and context_class_sampling_rate:
+            return calculate_normalized_hash(context_id) <= context_class_sampling_rate
+        return True
```

Comment:
an important behavior is missing here - we need another param to set the default sampling rate. We need new classes that don't appear in the config to allow sampling as well (and we'll even use a default of 0.001 for one of our customers). 

Right now you check if the dict was given (line 351 of this revision) to decide if it should get sampled. Please add another param for the default sampling rate, and if it's under 1 you should allow sampling (L354); and here use that default instead of returning True